packages/core/src/Keymap.ts

```diff
@@ -66,7 +74,13 @@ export class Keymap {
         });
         const maxPriority = Math.max(...matchingMappings.map(x => x.priority));
         const prioritaryMappings = matchingMappings.filter(x => x.priority === maxPriority);
-        return prioritaryMappings.map(mapping => mapping.boundCommand);
+        // undefined boundCommand means that we want to remove the shortuct.
+        const lastIndexUndefined = prioritaryMappings
+            .map(mapping => mapping.boundCommand)
+            .lastIndexOf(undefined);
+        return prioritaryMappings
+            .slice(lastIndexUndefined + 1)
+            .map(mapping => mapping.boundCommand);
```

Comment:
I don't understand this. Now that I think about it, what is the point of triggering two commands for one shortcut ? I think I would change `this._shortcutItems.filter` above by `this._shortcutItems.find` such that it returns the "latest registered" matching one. This way, if the latest matching one is the `undefined` one ? Fine, don't trigger anything. That could even remove the need for `priority` altogether if we have multiple "layers" of shortcuts like we discussed yesterday. Two keymaps. One for plugins, one for user config. Look for a match in user config and if not found look into the plugins one. Use "last registered first" as tie-breaker, which means you can use `find` rather than `filter` above and stop as soon as you find a match. If the match you found is an "undefined" one, you just found a shortcut that was disabled, do not trigger anything. I think it would be a lot cleaner and easier to understand that it is now.